exporter/trace/trace_proto.go

```diff
@@ -60,25 +60,33 @@ const (
 
 var userAgent = fmt.Sprintf("opentelemetry-go %s; cloudtrace-exporter %s", opentelemetry.Version(), version)
 
-func protoFromSpanData(s *export.SpanData, projectID string) *tracepb.Span {
+func generateDisplayName(s *export.SpanData, format DisplayNameFormatter) (displayName string) {
+	if format == nil {
+		switch s.SpanKind {
+		// TODO(ymotongpoo): add cases for "Send" and "Recv".
+		default:
+			displayName = fmt.Sprintf("Span.%s-%s", s.SpanKind, s.Name)
+		}
+	} else {
+		displayName = format(s)
+	}
```

Comment:
It's always a little easier to read code if you minimize nesting imo:

```suggestion
    if format != nil {
        return format(s)
    }

    switch s.SpanKind {
        // TODO(ymotongpoo): add cases for "Send" and "Recv".
        default:
            return fmt.Sprintf("Span.%s-%s", s.SpanKind, s.Name)
    }
```